web/src/components/__tests__/refresh-dropdown.spec.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+
+import RefreshDropdown from "../refresh-dropdown";
+
+describe("<RefreshDropdown />", () => {
+  const props = {
+    interval: null,
+    setInterval: jest.fn(),
+  }
+  it('should render component', () => {
+    const wrapper = shallow(
+      <RefreshDropdown {...props} />
+    );
+    expect(wrapper.find(RefreshDropdown)).toBeTruthy();
```

Comment:
Hmmm maybe it's a too obvious test. Is there any mean to test that e.g.:
- if a given time scale is selected, the state is forwarded somewhere (e.g. a mock),
- if the refresh button is clicked, some mock refresh action is triggered
..?

Reply:
Yes we still have some issues with jest + enzyme with click events.

I'm working on it but it may come in another PR